app/views/root/index.html.erb

```diff
@@ -48,9 +48,6 @@
 
       <section class="services-block" aria-labelledby="services-and-information-label" id="services-and-information">
         <div class="inner-block floated-children">
-          <div class="two-column-heading">
-            <h2 id="services-and-information-label">Services and information</h2>
```

Comment:
Oh and another thing, if we delete it we need to remove the [`aria-labelledby` attribute on line 49](https://github.com/alphagov/frontend/pull/751/files#diff-ffd72ca7720288aa96290dd748c1dfe1R49).